app/indexers/identity_metadata_indexer.rb

```diff
@@ -13,12 +13,13 @@ def to_solr
 
     {
       'objectType_ssim' => [object_type],
-      'dor_id_tesim' => [source_id_value, barcode, catkey].compact,
+      'dor_id_tesim' => [source_id_value, barcode, catkey, folio_instance_hrid].compact,
```

Comment:
for SEARCHING (tesim) it would be optimal to allow multiple values;  for the single value, we should be checking for ... refresh true among the multiple values??  I will make a separate ticket and ask Andrew and Arcadia.